client/public/index.html

```diff
@@ -3,6 +3,12 @@
   <head>
     <meta charset="utf-8" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link
```

Comment:
Duplicate. You should choose which one you prefer: either locally, either by using an external CDN like here